onwuvic/MyDiary
# #158910063 User Profile Reminder settings

#### What does this PR do?
Under user profile, a user should be able to set if they want a reminder or not.

#### Description of Task to be completed?
As a User, I should be able to check a button under settings, so that I can decide if I want a daily reminder or not

#### How should this be manually tested?
Given a user
When A user navigates to user profile settings
Then User should see a checkbox  to check

#### What are the relevant pivotal tracker stories?
#158910063
